### DISCUSSION TITLE
Does feed_list support sitemap?

### DISCUSSION THREAD

**Discussion by PttCodingMan** [gautamkrishnar/blog-post-workflow#191] [Q&A] [answered]
I was wondering if the feed_list supports sitemaps.
Would you be able to confirm this for me?

**Comment by gautamkrishnar** [comment, marked as answer]
nope, Sitemaps are just XML with https://www.sitemaps.org/protocol.html , it doesn't follow [RSS](https://en.wikipedia.org/wiki/RSS) standards.
